bindings/python/iota_sdk/utils.py

```diff
@@ -97,24 +97,60 @@ def compute_alias_id(output_id: OutputId) -> HexStr:
             'outputId': output_id
         })
 
+    @staticmethod
+    def compute_foundry_id(alias_id: str, serial_number: int, token_scheme_kind: int) -> HexStr:
+        """Computes the foundry id.
+        """
+        return _call_method('computeNftId', {
```

Comment:
```suggestion
        return _call_method('computeFoundryId', {
```

Reply:
good find! why didnt explorer team find this 😠